### PR TITLE
Correct doc to enable rendaring to URL (SOLVE #905)

### PR DESCRIPTION
**Pull request by yoshida-ryuhei**
- [x] I have added a related issue number.
- [x] I have read the CONTRIBUTING document on the Qiskit website. (Link - https://qiskit.org/documentation/contributing_to_qiskit.html#translating-documentation)
- [x] I have added my name to the LOCALIZATION_CONTRIBUTORS file under the language that I'd like to contribute.

### Summary
Solve the Issue #905.

Related #905.

### CODE REVIEW

**Comment by SooluThomas** [comment]
Hi @yoshida-ryuhei, we only edit the .po files in Crowdin. So the update that you proposed in this PR for the file docs/locale/ja_JP/LC_MESSAGES/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.po cannot be merged.

File: docs/locale/ja_JP/LC_MESSAGES/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.po [hunk 1]
```diff
@@ -20,7 +20,7 @@ msgstr ""
 
 #: ../../tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.ipynb:10
 msgid "This page was generated from `tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.ipynb`__."
-msgstr "当ページは `tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.ipynb`__から生成されました。"
+msgstr "当ページは `tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.ipynb`__ から生成されました。"
```
Comment (SooluThomas) [inline docs/locale/ja_JP/LC_MESSAGES/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.po:23]: revert the changes made to this file before approving or merging

```suggestion
msgstr "当ページは `tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.ipynb`__から生成されました。"
```

**Comment by yoshida-ryuhei** [comment]
Thank you for your quick reply.
I am going to add my name to `LOCALIZATION_CONTRIBUTORS` and make the change from Crowdin.
